custom_md_loader/classes/MetadataObjectCreator.cls

```diff
@@ -0,0 +1,279 @@
+/* 
+ * Copyright (c) 2016, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license. 
+ * For full license text, see LICENSE.txt file in the repo root  or https://opensource.org/licenses/BSD-3-Clause
+ */
+ 
+public with sharing class MetadataObjectCreator {
```

Comment:
We talked about either finding another way to accomplish this, or skipping custom object/field creation for now.

Reply:
I feel its a good feature to have. I am converting object/field creation using MetadataService.